CHANGELOG.md

```diff
@@ -50,6 +52,57 @@
 
 Although every attempt has been made to make this information as accurate as possible, please know there may be things that are omitted, not fully developed yet, or updates since this publication that were not included in the information below. Only the most pressing or significant items have been listed. For the entire list of tickets and or specific information about any given item, please visit the list at [Hyperleder Indy's Jira](https://jira.hyperledger.org/). Once logged in, simply navigate to Projects > Indy.
 
+## 1.11.0
+### Release date: Oct 31th, 2019
+
+### Component Version Information
+| Components | Version Numbers |
+| --- | --- |
+| indy-plenum | 1.11.0 |
+| indy-node | 1.11.0 |
+| sovrin | 1.1.60 |
+
+### Additional Information:
+**There are possible OOM issues during 3+ hours of target load or large catch-ups at 8 GB RAM nodes pool so 32 GB is recommended.**
+**Please be careful with demoting/promoting/adding nodes (see Known Issues for details).**
+
+### Major Changes
+- Switch to PBFT View Change protocol
+- Stability fixes
+
+### Detailed Changelog
+
+#### Major Fixes
+| Description | Additional Information | Ticket Number |
+| --- | --- | --- |
+| After a ledger-triggered upgrade of the Sovrin package failed, indy-node upgraded to an incorrect version | | [INDY-2161](https://jira.hyperledger.org/browse/INDY-2161) |
```

Comment:
We didn't do anything in INDY-2161, so I would not mention this ticket in release notes at all (especially as the first item)